rise-storage.html

```diff
@@ -1330,8 +1330,7 @@
         decodedUrl = url.substring(0, end !== -1 ? end : url.length);
       }
 
-      // Decode again to handle double slash encodings
```

Comment:
@rodrigopavezi I'm not sure what this comment means. Do you have an idea of what a test URL for this would look like?